### PR TITLE
 full view of download percent/speed

### PR DESCRIPTION
**Pull request by pwrusr**
on some screens while downloading files with larger content name (text) makes an incomplete view of
download percent/speed ........ A try to fix that with 4e55ab9

### CODE REVIEW

**Comment by pwrusr** [comment]
It was a mistake that all my commits got uploaded..please consider Update dimens.xml
Which I made changes on 9 Jan

**Comment by sargijeh** [comment]
He means something like this situation :)

File: app/src/main/java/tk/wasdennnoch/androidn_ify/systemui/notifications/NotificationHooks.java [hunk 1]
```diff
@@ -555,11 +555,19 @@ protected Object replaceHookedMethod(MethodHookParam param) throws Throwable {
...
             if (mAccentColor == 0) {
                 //noinspection deprecation
                 mAccentColor = context.getResources().getColor(context.getResources().getIdentifier("notification_icon_bg_color", "color", PACKAGE_ANDROID));
             }
             int c = mAccentColor;
+            try {
+                Drawable appIcon = context.getPackageManager().getApplicationIcon(packageName);
+                c = tk.wasdennnoch.androidn_ify.utils.ColorUtils.generateColor(appIcon, mAccentColor);
+                mGeneratedColors.put(packageName, c);
+                } catch (PackageManager.NameNotFoundException ignore) {
```
Comment (aviraxp) [inline app/src/main/java/tk/wasdennnoch/androidn_ify/systemui/notifications/NotificationHooks.java]: bracket

Comment (pwrusr) [inline app/src/main/java/tk/wasdennnoch/androidn_ify/systemui/notifications/NotificationHooks.java]: catch closed

**Comment by yzyboost** [comment]
Merge? Hahaha

**Comment by pwrusr** [comment]
@yzyboost  whats soo funny in this ?????

**Comment by Maxr1998** [comment]
@pwrusr you need to stash your previous changes. A rebase would be an idea here.

**Comment by pwrusr** [comment]
@Maxr1998 deleting would be nice idea (;
thank you

**Pull request closed without merging**